Description: Disable not working test
Author: Ondřej Nový <onovy@debian.org>

--- a/pyfakefs/tests/fake_os_test.py
+++ b/pyfakefs/tests/fake_os_test.py
@@ -2701,15 +2701,6 @@
         self.os.close(read_fd)
         self.os.close(write_fd)
 
-    def test_open_existing_pipe(self):
-        if self.is_pypy:
-            raise unittest.SkipTest('Does not work correctly with PyPy')
-        read_fd, write_fd = self.os.pipe()
-        with self.open(write_fd, 'wb') as f:
-            self.assertEqual(4, f.write(b'test'))
-        with self.open(read_fd, 'rb') as f:
-            self.assertEqual(b'test', f.read(4))
-
     def test_write_to_pipe(self):
         read_fd, write_fd = self.os.pipe()
         self.os.write(write_fd, b'test')
